src/main/java/uk/gov/register/presentation/view/ViewFactory.java

```diff
@@ -65,7 +67,11 @@ public EntryView getEntryView(Entry entry) {
         return new EntryView(requestContext, getCustodian(), getBranding(), entry);
     }
 
-    public EntryListView getEntryListView(List<Entry> entries, Pagination pagination) {
+    public EntryListView getEntryListView(List<Entry> entries, IPagination pagination) {
+        return new EntryListView(requestContext, pagination, getCustodian(), getBranding(), entries);
+    }
+
```

Comment:
Why do we need this method now when we also have getEntriesView, which is the same except that it takes a Collection instead of a List? Can we use getEntriesView in both places and remove getEntryListView?